src/naksu/mebroutines/mebroutines.go

```diff
@@ -345,7 +345,7 @@ func CopyFile(src, dst string) (err error) {
 
 // IfIntlCharsInPath returns true if path contains non-ASCII characters
 func IfIntlCharsInPath(path string) bool {
-	matched, err := regexp.MatchString("[^a-zA-Z0-9_\\-\\/\\:\\\\ ]", path)
+	matched, err := regexp.MatchString("[^a-zA-Z0-9_\\-\\/\\:\\\\ \\.]", path)
```

Comment:
I'd use raw literal strings here (https://stackoverflow.com/questions/46917331/what-is-the-difference-between-backticks-double-quotes-in-golang?rq=1), since they reduce the number of `\` escapes that are needed

I tested `` `[^a-zA-Z0-9_\-\/\:\\ \.]\` `` in https://regex101.com/ and it is a tad easier to read